src/Dialect/ONNX/ONNXOps.td.inc

```diff
@@ -6980,6 +6981,7 @@ def ONNXOneHotEncoderOp:ONNX_Op<"OneHotEncoder",
       return {7};
     }
   }];
+  let verifier = [{ return ::verify(*this); }];
```

Comment:
At the risk of repeating myself - this form will become obsolete and we'll have to modify all the code to remove it if it gets checked in like this. Use `hasVerifier` instead. There are examples in the tree of how to do this.

Reply:
Here's a change that did this as well: https://github.com/onnx/onnx-mlir/pull/1195

Reply:
How is updating the script related to this current pr? I don't understand.

Reply:
You happen to have stumbled into something that needs to be updated. Since you are making a change to the relevant script to add more outdated code, you would ideally also update the script and clean up the other usages. It is not needed to add the functionality you want to add, however, it is good practice.

Reply:
What mentioned that I would have to change the function header names? What do I change them to?

Reply:
You can look at the change I linked, but instead of being static functions, the `verify` functions instead belong to the class now.